integration/hyper_test.go

```diff
@@ -404,3 +432,30 @@ func (s *TestSuite) TestPauseAndUnpausePod(c *C) {
 	err = s.client.RemovePod(pod)
 	c.Assert(err, IsNil)
 }
+
+func (s *TestSuite) TestGetPodStats(c *C) {
```

Comment:
`GetPodStats` only works for `libvirt`, but the tests all cover `qemu` hypervisor. Could you make this test only runs for `libvirt`? 

The hypervisor type could be got by calling `Info` api:

```
hyperctl info | grep Execu
Execution Driver: libvirt
```